variables.tf

```diff
@@ -179,4 +200,19 @@ variable "verify_worker_network_readiness" {
   default     = true
 }
 
+variable "addons" {
+  type = object({
+    alb-oauth-proxy           = optional(string)
+    debug-tool                = optional(string)
+    image-key-synchronizer    = optional(string)
+    istio                     = optional(string)
+    openshift-data-foundation = optional(string)
+    static-route              = optional(string)
+    cluster-autoscaler        = optional(string)
+    vpc-block-csi-driver      = optional(string)
+  })
```

Comment:
@Aashiq-J Have we tested these add ons? How is one supposed to enable them? They seems to be a string type, I would of thought bool?

Reply:
@ocofaigh 
https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/blob/main/examples/single_zone_cluster/main.tf
This example showcase the use of addons, the user need to pass the list of addons along with the version to be installed on the cluster.

Reply:
I think we need to make to update the example name so its obvious that it shows how to enable auto scaler 
Also this will need to be exposed now in https://github.com/terraform-ibm-modules/terraform-ibm-ocp-all-inclusive now right (and internal one too)

Reply:
The default value for list of addons is null, so its not a mandatory variable to pass. When it is passed as null only the default vpc-driver is installed.

Reply:
https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/pull/161